### ISSUE TITLE
Include Collector-Sidecar Configurations in Content Pack

### ISSUE THREAD

**Issue by jensschulze** [Graylog2/graylog2-server#2642]
Feature Request
## Expected Behavior

For the sake of simplicity and consistency the configurations for the Graylog Collector Sidecar should be integrated into the Content Packs.
## Current Behavior

Whenever I spin up a new Graylog Server I have to configure the Graylog Collector Sidecar(s) manually, or have to fumble around with MongoDB.

**Comment by awilkins** [comment]
I'm doing this with the "uri" task in Ansible : 

```yaml
  - name: Create sidecar collector config
    register: apply
    uri:
      url: "http://localhost:9000/api/plugins/org.graylog.plugins.collector/configurations"
      method: POST
      body: "{{ lookup('file', 'collector-config.json') }}"
      body_format: "json"
      user: admin
      HEADER_Accept: "application/json"
      password: "{{ admin_pass }}"
      status_code: 200
    when: bundle.status == 404
    changed_when: yes
```

**Comment by jensschulze** [comment]
There are quite a few ways to get the Collector config into Graylog (I'm using a script that makes the respective API call as soon as the server response is "ALIVE").
I think that there is at least one big point speaking against putting the Collector config into a Content pack: The Collector config is tightly coupled with your setup whilst a Content pack rather contains type definitions and their respective configuration. But anyway it should be possible to save the Collector config as a file that can be ingested at Graylog start, just like a Content pack.

**Comment by edmundoa** [comment]
Closing, as this is already implemented and will be part of 3.0.0.
